FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -5,6 +5,24 @@
 import Foundation
 
 public final class RemoteFeedLoader: FeedLoader {
+	struct FeedImageStruct: Codable {
```

Comment:
Following from my previous comment, the same applies to `FeedImageStruct`. The way it's being used at this moment it should be marked as `private`, but if you decide to reference this type in an extension (from one of my previous suggestions), you might also consider `fileprivate`.
```suggestion
	private struct FeedImageStruct: Decodable {
```